docs/en/observability/aws-deploy-elastic-serverless-forwarder.asciidoc

```diff
@@ -381,14 +372,14 @@ If you chose this option, you have to:
 [[sample-direct-publish-config-file]]
 === Create `publish-config.yaml` for the publishing script
 
-Publishing Elastic Serverless Forwarder directly requires a `publish-config.yaml` file to be passed as argument in order to <<aws-serverless-forwarder-run-publish-script>>.
+To deploy the forwarder directly, you need to define a `publish-config.yaml` file and pass this as an argument in the <<aws-serverless-forwarder-run-publish-script, publishing script>>.
 
 Save the following YAML content as `publish-config.yaml` and edit as required before running the publishing script. You should remove any inputs or arguments you are not using.
 
 [source, yaml]
 ----
 
-kinesis-data-stream:
+kinesis_data_stream:
```

Comment:
`kinesis-data-stream` is the proper syntax
probably a leftover since you started the review and I pushed the latest change